### PR TITLE
A few more Ramda lint rules

### PR DESCRIPTION
**Pull request by jtbandes**
**User-Facing Changes**
None

**Description**
- `R.keys/values` -> `Object.keys/values`
- `R.find` -> `Array#find`
- `R.all/any` -> `Array#every/some`

Relates to FG-4805

### CODE REVIEW

File: packages/studio-base/src/components/Chart/datasets.ts [hunk 1]
```diff
@@ -85,13 +85,13 @@ export function* iterateTyped<T extends { [key: string]: Array<any> | Float32Arr
...
     // Find a property for which we can check the length
-    const first = R.head(R.values(slice));
+    const first = R.head(Object.values(slice));
     if (first == undefined) {
       continue;
     }
 
     for (let j = 0; j < first.length; j++) {
-      for (const key of R.keys(slice)) {
+      for (const key of Object.keys(slice) as (keyof typeof slice)[]) {
```
Comment (defunctzombie) [inline packages/studio-base/src/components/Chart/datasets.ts:94]: Why the cast?

**Review by defunctzombie — APPROVED** [review]
What was the genesis of this? Did we decide that using the builtins is more idiomatic?

File: packages/studio-base/src/components/Chart/datasets.ts (hunk 1 above)
Comment (jtbandes) [reply packages/studio-base/src/components/Chart/datasets.ts:94]: `Object.keys` has poor TS types :( https://www.mattstobbs.com/object-keys-typescript/

**Comment by jtbandes** [comment]
It is my small attempt to hold back the flood of Ramda everywhere  (and I don't believe there's any reason to use the Ramda ones over the builtin ones? seems like they could only be slower?)

File: packages/studio-base/src/components/Chart/datasets.ts (hunk 1 above)
Comment (cfoust) [reply packages/studio-base/src/components/Chart/datasets.ts:94]: lol, this begs the question of why we're banning R.keys then, right?

**Comment by cfoust** [comment]
Is this something we've decided on?

**Comment by jtbandes** [comment]
Nope it was just one of the things I noted down on my wishlist in FG-4805. Feel free to debate it here

File: packages/studio-base/src/components/Chart/datasets.ts (hunk 1 above)
Comment (jtbandes) [reply packages/studio-base/src/components/Chart/datasets.ts:94]: We could exclude this one if it seems like a significant benefit (or we could make the lint rule go the other way and enforce `R.keys` over `Object.keys`). Will note that the repo currently has 46 uses of `Object.keys` and 3 uses of `R.keys`.

Comment (defunctzombie) [reply packages/studio-base/src/components/Chart/datasets.ts:94]: We could override the TS types for Object.keys like we do for json stringify.

**Comment by defunctzombie** [comment]
I like the idea of using the builtin language functions where available and don't know why I'd turn to the ramda variants for that. I would turn to ramda for its other features. Though I would also ask what it does that lodash does not? Maybe it has more functions so we should use it instead of lodash? Or do the two offer different things?

**Comment by jtbandes** [comment]
Merging as-is for now since this is a path to consistency with relatively minimal changes. If someone else wants to champion some Object.keys type overrides or migrating to R.keys everywhere, I would be open to that as well. We've learned that both Ramda and TS are kind of lying with their type definitions here – pick your poison.

**Pull request merged**